src/library/scala/StringContext.scala

```diff
@@ -91,7 +91,7 @@ case class StringContext(parts: String*) {
    *  @note   The Scala compiler may replace a call to this method with an equivalent, but more efficient,
    *          use of a StringBuilder.
    */
-  def s(args: Any*): String = standardInterpolator(processEscapes, args, parts)
+  def s(args: Any*): String = standardInterpolator(processEscapes, args, parts.toSeq)
```

Comment:
ah right, this method is now a fast-tracked macro, although it was already special-cased in the compiler (previously in refchecks)

Reply:
that's probably a red herring, though -- I think the real problem is in `caseModuleUnapplyMeth`